digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java

```diff
@@ -530,9 +532,31 @@ RunTaskRequest buildRunTaskRequest(
         setEcsTaskStartedBy(clientConfig, runTaskRequest);
         setEcsNetworkConfiguration(clientConfig, runTaskRequest);
         setCapacityProviderStrategy(clientConfig, runTaskRequest);
+        setPlacementStrategy(clientConfig, runTaskRequest);
         return runTaskRequest;
     }
 
+    private void setPlacementStrategy(EcsClientConfig clientConfig, RunTaskRequest runTaskRequest) throws ConfigException
+    {
+        if (clientConfig.getPlacementStrategyType().isPresent()) {
+            final PlacementStrategyType placementStrategyType;
+            try {
+                placementStrategyType = PlacementStrategyType.fromValue(clientConfig.getPlacementStrategyType().get());
+            }
+            catch (IllegalArgumentException invalidReason) {
```

Comment:
[minor] `invalidReason` sounds a bit weird since this is a just exception here. If you extracted a pure reason-like object from an exception like with `Reason invalidReason = ex.getReason()`, it would be good.

Reply:
What I meant was just "This is an `IllegalArgumentException` object and the actual reason is stored in this exception object's message". But surely it is wired a bit. I will rename the exception object and leave a comment to suggest it.